File: 2-semestre/uri-online-judge/javascript/1173.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let arr = [];
const n = parseInt(lines.shift());

arr[0] = n;

for (let i = 1; i < 10; i++) {
    arr[i] = arr[i-1] * 2;
}

for (let j = 0; j < 10; j++) {
    console.log(`N[${j}] = ${arr[j]}`);
}

File: 2-semestre/uri-online-judge/c/1173.c
#include <stdio.h>
 
int main() {    
    int arr[10], n, i, j;

    scanf("%d", &n);

    arr[0] = n;

    for (i = 1; i < 10; i++) {
        arr[i] = arr[i-1] * 2;
    }

    for (j = 0; j < 10; j++) {
        printf("N[%d] = %d\n", j, arr[j]);
    }

    return 0;
}
